main.tf

```diff
@@ -1,73 +1,40 @@
 # Using tf_admin module
 
-# Apply the tf_label module for this resource
-module "label" {
-  source    = "git::https://github.com/cloudposse/tf_label.git?ref=tags/0.1.0"
-  namespace = "${var.namespace}"
-  stage     = "${var.stage}"
-  name      = "${var.name}"
+resource "aws_security_group_rule" "ssh" {
+  from_port         = 22
+  protocol          = "-1"
+  security_group_id = "${module.instance.security_group.id}"
+  to_port           = 22
+  type              = "ingress"
+  cidr_blocks       = ["0.0.0.0/0"]
 }
 
-resource "aws_security_group" "default" {
-  name        = "${module.label.id}-ssh"
-  vpc_id      = "${var.vpc_id}"
-  description = "Allow SSH access from any source"
-
-  tags {
-    Name      = "${module.label.id}"
-    Namespace = "${var.namespace}"
-    Stage     = "${var.stage}"
-  }
-
-  ingress {
-    protocol  = "tcp"
-    from_port = 22
-    to_port   = 22
-
-    cidr_blocks = [
-      "0.0.0.0/0",
-    ]
-  }
-
-  egress {
-    protocol  = "-1"
-    from_port = 0
-    to_port   = 0
-
-    cidr_blocks = [
-      "0.0.0.0/0",
-    ]
-  }
-
-  lifecycle {
-    create_before_destroy = true
-  }
-}
 
-# Using tf_instance module
+# Using terraform-aws-ec2-instance module
 module "instance" {
-  source              = "git::https://github.com/cloudposse/tf_instance.git?ref=tags/0.3.5"
-  namespace           = "${var.namespace}"
-  name                = "${var.name}"
-  stage               = "${var.stage}"
-  vpc_id              = "${var.vpc_id}"
-  subnets             = "${var.subnets}"
-  ansible_arguments   = "${var.ansible_arguments}"
-  ansible_playbook    = "${var.ansible_playbook}"
-  ansible_envs        = "${var.ansible_envs}"
-  ssh_key_pair        = "${var.ssh_key_pair}"
-  github_api_token    = "${var.github_api_token}"
-  github_organization = "${var.github_organization}"
-  github_team         = "${var.github_team}"
-  instance_type       = "${var.instance_type}"
+  source                        = "git::https://github.com/cloudposse/terraform-aws-ec2-instance.git?ref=tags/0.3.8"
+  namespace                     = "${var.namespace}"
+  name                          = "${var.name}"
+  stage                         = "${var.stage}"
+  vpc_id                        = "${var.vpc_id}"
+  subnets                       = "${var.subnets}"
+  ansible_arguments             = "${var.ansible_arguments}"
+  ansible_playbook              = "${var.ansible_playbook}"
+  ansible_envs                  = "${var.ansible_envs}"
+  ssh_key_pair                  = "${var.ssh_key_pair}"
+  github_api_token              = "${var.github_api_token}"
+  github_organization           = "${var.github_organization}"
+  github_team                   = "${var.github_team}"
+  instance_type                 = "${var.instance_type}"
+  create_default_security_group = "${var.create_default_security_group}"
```

Comment:
We should hardcode this option to `true` because it is required for `aws_security_group_rule` https://github.com/cloudposse/terraform-aws-ec2-admin-server/pull/16/files#diff-7a370d8342e7203b805911c92454f0f4R6